mlflow/server/js/src/components/ExperimentPage.js

```diff
@@ -28,6 +28,8 @@ class ExperimentPage extends Component {
         paramKeyFilterString: urlState.params === undefined ? "" : urlState.params,
         metricKeyFilterString: urlState.metrics === undefined ? "" : urlState.metrics,
         searchInput: urlState.search === undefined ? "" : urlState.search,
+        orderByKey: urlState.orderByKey === undefined ? null : urlState.orderByKey,
+        orderByAsc: urlState.orderByAsc === undefined ? true : urlState.orderByAsc,
```

Comment:
Replacing this with `orderByAsc: urlState.orderByAsc === undefined ? true : urlState.orderByAsc === 'true'` fixes the warning in ^